README.md

```diff
@@ -1,66 +1,31 @@
 
-`npm install nylas` or `yarn add nylas`
+# Table of Contents
 
+* [Install](#install)
+* [Usage](#usage)
+* [Quick Start](#quick-start)
+* [Contributing](#contributing)
 
-API Overview
-------------
+# Install
 
-Every resource (i.e., messages, events, contacts) is accessed via an instance of `Nylas`. Before making any requests, be sure to call `config` and initialize the `Nylas` instance with your `appId` and `appSecret`. Then, call `with` and pass it your `accessToken`. The `accessToken` allows `Nylas` to make requests for a given account's resources.
+Install the SDK using one of the following commands:
 
-```javascript
-const Nylas = require('nylas');
-
-Nylas.config({
-  appId: APP_ID,
-  appSecret: APP_SECRET,
-});
-
-const nylas = Nylas.with(ACCESS_TOKEN);
+```shell
+npm install nylas
 ```
 
-Every resource method accepts an optional callback as the last argument:
+or
 
-```javascript
-nylas.threads.list({}, (err, threads) => {
-  console.log(threads.length);
-});
+```shell
+yarn add nylas
 ```
 
-Additionally, every resource method returns a promise, so you don't have to use callbacks if your code is promise-friendly. Here's an example using promises:
+# Usage
 
-```javascript
-nylas.threads.list({}).then(threads => {
-  console.log(threads.length);
-});
-```
-
-And here's an example using async/await:
-
-```javascript
-const getThreadCount = async nylas => {
-  const threads = await nylas.threads.list({});
-  return threads.length;
-};
-```
-
-
-Authentication
------
-The Nylas REST API uses server-side (three-legged) OAuth, and the Node.js bindings provide convenience methods that simplify the OAuth process. For more information about authenticating users with Nylas, visit the [API docs](https://nylas.com/docs/#authentication).
-
-`urlForAuthentication()` takes in an `options` object, which must have a `redirectURI` property defined. Other supported, but optional, properties are:
-
-* `loginHint` - The user's email address, if known.
-* `state` - An arbitrary string that will be returned back as a query param in your `redirectURI`.
-* `scopes` - An array of which scopes you'd like to auth with. The Nylas API provides granular authentication scopes that empower users with control over what level of access your application has to their data. See supported [Authentication Scopes](https://docs.nylas.com/docs/authentication-scopes) for a full list of scopes and details behind the scopes. If omitted, defaults to all scopes.
-
-### Step 1: Redirect the user to Nylas
+Every resource (i.e. messages, events, contacts, etc.) is accessed via an instance of ```Nylas```. Before making any requests, call ```config``` and initialize the Nylas instance with your APP ID and APP Secret. Then, call ```with``` and pass it your access token. The access token allows Nylas to make requests for a given email account's resources.
```

Comment:
Unfortunately App ID and App Secret are not consistent with the naming in our API docs, which uses `client_id` and `client_secret`. We may want to make a quick note here to note the distinction (and that they are equivalent). Eventually we'll release a new SDK version that uses `client_id` instead.

Reply:
@pfista This is resolved, as well as the other comments.